src/schemas/activity.schema.ts

```diff
@@ -0,0 +1,16 @@
+  
+import { object, string, ref, number } from 'yup';
+
+export const createActivitySchema = object({
+  body: object({
+    label: string()
+    .required(`Label is required`)
+    .min(3, `Your label  is to short - should be 3 chars minimum`)
+    .max(100, `Your label is to long - should be 100 chars maximum`)
+    .matches(/^[a-zA-Z0-9_.-]*$/, `Label can only contain Latin letters.`),
+    clientMax: number(),
```

Comment:
Maybe you can add a min restriction at 1. Actually, it's possible to add -1 or -2 client max.